MASH-FRET/source/traces/processing/cross-talk/gammaCorr.m

```diff
@@ -17,23 +17,44 @@
     % collect molecule traces
     I_den = p.proj{proj}.intensities_denoise(:, ...
         ((mol-1)*nChan+1):mol*nChan,:);
+    I_DTA = p.proj{proj}.intensities_DTA(:, ...
+        ((mol-1)*nChan+1):mol*nChan,:);
+    
+    % set method to "manual" if no intensity discretization is found
+    if sum(isnan(I_DTA))
```

Comment:
Currently the "if" statement returns a non-zero-value whenever NaNs are found (i.e. when the trace is not discretized at all) and thus resets to "manual". But if the traces is only cut, then those values are also NaNs. Therefore I suggest to change the line to 
```
all(isnan(I_DTA))
```
This checks if all values are NaNs (i.e. no discretization of the intensity trace)

Reply:
I agree, thank you for the suggestion!